setup.py

```diff
@@ -11,6 +11,13 @@
             " install setuptools).")
     sys.exit(1)
 
+with open('requirements.txt') as requirements_file:
```

Comment:
setup.py shouldn't be checked for py2.4 support.  It will only run on py2.6+ as it is controller only